execFileSystem.go

```diff
@@ -5,16 +5,19 @@ import (
 	"fmt"
 	"html/template"
 	"net/http"
-	"text/template/parse"
 )
 
 // FileSystemExecutor loads view templates as a path from a Go HTML template file.
 // The underlying file system is abstracted through the http.FileSystem interface to allow for
 // in-memory use.
+//
+// The optional KeyedString map will be checked before the loader attempts to use the FS
+// instance when obtain a template string
 type FileSystemExecutor struct {
-	FS    http.FileSystem
-	Funcs template.FuncMap
-	exec  Executor
+	FS          http.FileSystem
+	Funcs       template.FuncMap
+	KeyedString map[string]string
```

Comment:
Would it make sense for vanilla FileSystemExecutor to remain, and a new `HybridFileSystemExecutor` created?

Or you would prefer not to have the Hybrid variant of each Executor type?

Reply:
I think that makes more sense in principle than in practice. It is more likely that the other executors are never used and that the FS exec is the only useful one for a non-basic use case.

The executors are definitely more cumbersome than other parts of the library.  The main goal is performance and then extensibility, so that users can initialize templates their own way. I didn't find a simpler mechanism to achieve this.